### PR TITLE
Android: Implement Messages view

### PR DESCRIPTION
**Pull request by nc-kano**

### CODE REVIEW

**Comment by nc-kano** [comment]
Screenshot_1605880283

**Comment by nc-kano** [comment]
Screenshot_1605881086

File: NDB.Covid19/NDB.Covid19/ViewModels/MessagesViewModel.cs [hunk 1]
```diff
@@ -11,9 +11,11 @@ public class MessagesViewModel
     {
         public static string MESSAGES_HEADER => "MESSAGES_HEADER".Translate();
         public static string MESSAGES_NO_ITEMS_TITLE => "MESSAGES_NOMESSAGES_HEADER".Translate();
+        public static string MESSAGES_NEW_MESSAGES_HEADER => "MESSAGES_NEW_MESSAGES_HEADER".Translate();
         public static string MESSAGES_NO_ITEMS_DESCRIPTION => "MESSAGES_NOMESSAGES_LABEL".Translate();
         public static string MESSAGES_LAST_UPDATED_LABEL => "MESSAGES_LAST_UPDATED_LABEL".Translate();
         public static string MESSAGES_ACCESSIBILITY_CLOSE_BUTTON => "MESSAGES_ACCESSIBILITY_CLOSE_BUTTON".Translate();
+        public static string MESSAGES_NEW_ITEM => "MESSAGES_NEW_ITEM".Translate();
```
Comment (pavshr) [inline NDB.Covid19/NDB.Covid19/ViewModels/MessagesViewModel.cs:18]: can you group this one and MESSAGES_NEW_MESSAGES_HEADER? so there is some structure e.g.,
MESSAGES_NEW_MESSAGES_HEADER
MESSAGES_NEW_ITEM
MESSAGES_NO_ITEMS_TITLE
MESSAGES_NO_ITEMS_DESCRIPTION

File: NDB.Covid19/NDB.Covid19.Droid/NDB.Covid19.Droid.csproj [hunk 2]
```diff
@@ -151,6 +151,7 @@
     <AndroidResource Include="Resources\drawable-xxhdpi\logo_fhi_blue.png" />
     <AndroidResource Include="Resources\drawable\calendar_border.xml" />
     <AndroidResource Include="Resources\drawable\logoKortBl.png" />
```
[the remark this answers is not in the document]
Comment (nc-kano) [reply NDB.Covid19/NDB.Covid19.Droid/NDB.Covid19.Droid.csproj:153]: no, this is new one

File: NDB.Covid19/NDB.Covid19.Droid/Views/Messages/MessagesActivity.cs [hunk 3]
```diff
@@ -75,22 +76,27 @@ private void CloseLocalNotification()
             notificationManagerCompat.Cancel(LocalNotificationsManager.NotificationId);
         }
 
-        private void Init()
+        private async void Init()
```
[the remark this answers is not in the document]
Comment (nc-kano) [reply NDB.Covid19/NDB.Covid19.Droid/Views/Messages/MessagesActivity.cs:79]: beacuse of await MessageUtils.GetAllUnreadMessages()

File: NDB.Covid19/NDB.Covid19/ViewModels/MessagesViewModel.cs (hunk 1 above)
Comment (nc-kano) [reply NDB.Covid19/NDB.Covid19/ViewModels/MessagesViewModel.cs]: done

**Review by matishadow — APPROVED** [review]

**Review by Frederik-S-N-dk — APPROVED** [review]

**Pull request merged**